src/systempolicy/systemPolicy.go

```diff
@@ -587,7 +587,6 @@ func checkIfMetadataMatches(pin types.KnoxSystemPolicy, hay []types.KnoxSystemPo
 	for idx, v := range hay {
 		if pin.Metadata["clusterName"] == v.Metadata["clusterName"] &&
 			pin.Metadata["namespace"] == v.Metadata["namespace"] &&
-			pin.Metadata["containername"] == v.Metadata["containername"] &&
```

Comment:
`Container Name` is specified by the user while deploying a resource to Kubernetes right? How is it going to change even if there is a restart? Isn't this value going to be the same up until the user changes it?

Reply:
Yes @vishnusomank 
This should be same, but this is varying in kubearmor logs.

Reply:
@rajaSahil With the current log/alert structure in kubearmor, I don't think we get UUID.